transformer_engine/pytorch/te_onnx_extensions.py

```diff
@@ -166,6 +166,16 @@ def onnx_fp8_relu(g, inputs, scale, amax, scale_inv, fp8_tensor, otype):
     return relu
 
 
+@symbolic_helper.parse_args("v", "v", "v", "fs", "i", "i")
+def onnx_fp8_srelu(g, inputs, scale, amax, scale_inv, fp8_tensor, otype):
+    """ONNX graph for fp8_srelu"""
+    # pylint: disable=unused-argument
+    srelu = compute_in_fp32(g, inputs, torch.onnx.symbolic_opset9.srelu)
```

Comment:
I don't think this exists: https://github.com/pytorch/pytorch/blob/main/torch/onnx/symbolic_opset9.py

Reply:
I remove srelu from onnx and it tests. Thanks